jobs/rocoto/anal.sh

```diff
@@ -4,9 +4,43 @@ source "${HOMEgfs}/ush/preamble.sh"
 
 ###############################################################
 # Source FV3GFS workflow modules
-. ${HOMEgfs}/ush/load_fv3gfs_modules.sh
-status=$?
-[[ ${status} -ne 0 ]] && exit ${status}
+#. ${HOMEgfs}/ush/load_fv3gfs_modules.sh
+#status=$?
+#[[ ${status} -ne 0 ]] && exit ${status}
+
+# TODO: clean this up
+#Source appropriate modulefiles based on machine
+
+source "${HOMEgfs}/ush/detect_machine.sh"
+if [[ "${MACHINE_ID}" = "wcoss2" ]]; then
+    # Source FV3GFS workflow modules
+    . "${HOMEgfs}"/ush/load_fv3gfs_modules.sh
+    status=$?
+    [[ ${status} -ne 0 ]] && exit "${status}"
+else
+    # Append compiler (only on machines that have multiple compilers)
+    COMPILER=${COMPILER:-"intel"}
+    if [[ "${MACHINE_ID}" = "hera" ]] || [[ "${MACHINE_ID}" = "cheyenne" ]]; then
+        MACHINE_ID=${MACHINE_ID}.${COMPILER}
+    fi
+
+    # Source machine specific GSI-EnKF modules
+    set +x
+    source "${HOMEgfs}/ush/module-setup.sh"
+    module use "${HOMEgfs}/sorc/gsi_enkf.fd/modulefiles"
+    module load gsi_"${MACHINE_ID}"
+
+    if [[ "${MACHINE_ID}" = "orion" ]]; then
+       module load miniconda3/4.12.0
+       set +u
+       conda activate regional_workflow_cmaq
+       set -u
```

Comment:
We have a function setup by the preamble to handle turning this back on. This way it isn't turned on if `$STRICT` was `"NO"`.
```suggestion
       set_strict
```